docs/source/_json/lock.resp

```diff
@@ -0,0 +1,12 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "creator": "admin", 
+  "locked": true, 
+  "name": "plone.locking.stealable", 
+  "stealable": true, 
+  "time": 1477076400.0, 
```

Comment:
@tisto I'm not super happy with the term "time" but I'm using the same term as in plone.locking for consistency reasons (see https://github.com/plone/plone.locking/blob/master/plone/locking/lockable.py#L150). I'm also ok to rename it to "modified" or "modification-time".